crawler/src/main/java/com/movie/recommender/crawler/dataimport/MovieDataInitializer.java

```diff
@@ -0,0 +1,44 @@
+package com.movie.recommender.crawler.dataimport;
+
+
+import com.movie.recommender.common.model.movie.MovieDetails;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class MovieDataInitializer {
+    private static final Logger logger = LoggerFactory.getLogger(MovieDataInitializer.class);
```

Comment:
again, we use `@Slf4j `annotations in other parts of the project
keep everything consistent